### PR TITLE
[DO NOT MERGE YET] Updated Simplified Chinese translation

### PR DESCRIPTION
**Pull request by User670**
**Do not merge until I say otherwise!** This edit is not complete; there are a few things that I am unsure of.

- Fishing: what correction? what "ticks ahead"?
- item group? kit? (For now I translated "item group" literally; "kit" as "preset inventory" as described in the change log that added that command)
- "Stopped sound '%s' on source '%s' for self", Chinese seems to have a different word order for the two %s. Can they swap positions somehow, or do I have to rephrase to keep the order? (also, what source?)

### CODE REVIEW

**Comment by haykam821** [comment]
You can open your pull request as a draft if it is not ready to be merged yet.

File: src/main/resources/assets/clientcommands/lang/zh_cn.json [hunk 1]
```diff
@@ -36,10 +50,73 @@
...
+  "commands.citemgroup.notFound": "道具组 \"%s\" 未找到",
+  "commands.citemgroup.outOfBounds": "数组下标 %d 越界",
+  "commands.citemgroup.saveFile.failed": "无法保存道具组文件",
+  "commands.citemgroup.addGroup.success": "成功添加道具组 \"%s\"",
+  "commands.citemgroup.addGroup.illegalCharacter": "道具组只能包含以下字符： [a-z0-9/._-]",
+  "commands.citemgroup.addGroup.alreadyExists": "道具组 \"%s\" 已经存在",
+  "commands.citemgroup.removeGroup.success": "成功移除道具组 \"%s\"",
+  "commands.citemgroup.addStack.success": "成功将 %s 添加到 \"%s\"",
+  "commands.citemgroup.removeStack.success": "成功从 \"%s\" 移除下标为 %d 的道具",
+  "commands.citemgroup.setStack.success": "成功从 \"%s\" 将下表为 %d 的道具更改为 %s",
```
Comment (0f-0b) [inline src/main/resources/assets/clientcommands/lang/zh_cn.json]:
```suggestion
  "commands.citemgroup.setStack.success": "成功从 \"%s\" 将下标为 %d 的道具更改为 %s",
```

File: src/main/resources/assets/clientcommands/lang/zh_cn.json [hunk 2]
```diff
@@ -36,10 +50,73 @@
...
+  "commands.cplaysound.success": "对自己播放声音 %s",
+
+  "commands.crelog.failed": "重新登录失败",
 
   "commands.crender.entities.success": "实体渲染规则已被更新",
 
+  "commands.cstopsound.success.sourceless.any": "为自己停止了所有声音",
+  "commands.cstopsound.success.sourceless.sound": "为自己停止了声音 '%s'",
+  "commands.cstopsound.success.source.any": "为自己停止了所有 '%s' 声音",
+  "commands.cstopsound.success.source.sound": "为自己停止了来自 '%s$2' 的 '%s$1' 声音",
```
Comment (0f-0b) [inline src/main/resources/assets/clientcommands/lang/zh_cn.json]:
```suggestion
  "commands.cstopsound.success.source.sound": "为自己停止了来自 '%2$s' 的 '%1$s' 声音",
```

**Review by 0f-0b — CHANGES_REQUESTED** [review]

**Review by 0f-0b — APPROVED** [review]

**Comment by User670** [comment]
Can I get some info on the questions I asked? Especially does the `%2$s` thing work?

**Comment by Earthcomputer** [comment]
- For rng fishing, the correction is the amount of milliseconds earlier the packet is sent to cancel out delay caused by high ping. "Ticks ahead" is how many ticks away from tick the fishing rod should have been pulled in, which roughly represents the correction the client needs to apply for next time.
- Item group is like a creative tab in the creative menu. A kit is like a preset inventory, yes.
- Yes, `%2$s` works, that's how you change the word order from English.

File: src/main/resources/assets/clientcommands/lang/zh_cn.json [hunk 3]
```diff
@@ -36,10 +50,73 @@
   "commands.cfinditem.starting.keepSearching": "正在持续搜索 %s。",
   "commands.cfinditem.total": "一共找到了 %d 个 %s 。",
 
+  "commands.cfish.addGoal.success": "成功添加目标 %s",
+  "commands.cfish.correctLoot": "获得了正确的掉落物， 修正值 %dms",
+  "commands.cfish.error.crackFailed": "破解钓鱼竿浮标随机数失败， 服务器是否安装了 Mods？",
+  "commands.cfish.error.failed": "钓鱼竿浮标配置不正确",
+  "commands.cfish.error.outOfWater": "钓鱼竿附表不在水中， 随机数操控无法工作",
```
Comment (Neubulae) [inline src/main/resources/assets/clientcommands/lang/zh_cn.json:57]: “浮标”，a typo

**Comment by Earthcomputer** [comment]
Why close the PR?

**Comment by User670** [comment]
> Why close the PR?

It's probably outdated, and I don't plan to work on it any more. If I want to keep working on it, I'll work on the fresh thing in the repo, not this.

**Comment by Neubulae** [comment]
lemme take a look and see what I can do with translation then, I'll do it when I have time

**Pull request closed without merging**